pals/alex.md

```diff
@@ -0,0 +1,68 @@
+# Alex Gozin #
+
+> "Who wouldn't want a good time?"
+
+#### Species, Class, Level: ####
+Human Paladin of Sehanine, Level 1
+
+#### Languages known: ####
+Common, Elven
+
+#### Attributes ####
+* Str: 14  (+2)
+* Con: 12  (+1)
+* Dex: 13* (+1)
+* Int: 10  (+0)
+* Wis: 13  (+1)
+* Cha: 16  (+3)
+
+\*+2 human bonus
```

Comment:
You can get a strictly better array if you put your bonus in CHA:

* Str: 14
* Con: 13
* Dex: 13
* Int: 10
* Wis: 14
* Cha: 14 -> +bonus to 16

I put the leftover 2 points into Con and Wis, but you might want to bump just Str or Wis a little more instead.